lib/lte_link_control/lte_lc.c

```diff
@@ -957,8 +978,13 @@ int lte_lc_edrx_req(bool enable)
 	if (enable) {
 		char edrx_req[25];
 
-		snprintf(edrx_req, sizeof(edrx_req),
-			 "AT+CEDRXS=2,%d,\"%s\"", actt, edrx_param);
+		if (strlen(edrx_param) == 4) {
+			snprintf(edrx_req, sizeof(edrx_req),
+				"AT+CEDRXS=2,%d,\"%s\"", actt, edrx_param);
+		} else {
+			snprintf(edrx_req, sizeof(edrx_req),
+				"AT+CEDRXS=1,%d", actt);
```

Comment:
Is there a reason to why eDRX is enabled without unsolicited results codes in this case?